### ISSUE TITLE
Error when init mm-snap

### ISSUE THREAD

**Issue by Vid201** [MetaMask/snaps-skunkworks#206]
Error when running `mm-snap init`

```
MetaMask Snaps: Initialize

package name: (example) 
version: (1.0.0) 
description: 
entry point: (index.js) 
test command: 
git repository: 
keywords: 
author: 
license: (ISC) 
About to write to /home/vid/Documents/Blockchain_LabUM/MetaMask/example/package.json:

{
  "name": "example",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}


Is this OK? (yes) yes

Init: Building 'snap.manifest.json'...

Use default Snap manifest values? (yes) yes
Using default values...

Init: Created 'snap.manifest.json'.
Init: Created 'index.js'.
Init: Created 'index.html'.
Init: Wrote 'snap.config.json' config file
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'babelify' from '/home/vid/Documents/Blockchain_LabUM/MetaMask/example'
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:146:35
    at processDirs (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:299:39)
    at isdir (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:306:32)
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:34:69
    at FSReqCallback.oncomplete (fs.js:169:21)
Emitted 'error' event on Browserify instance at:
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/browserify/index.js:375:34
    at /home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:148:17
    at processDirs (/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/resolve/lib/async.js:299:39)
    [... lines matching original stack trace ...]
    at FSReqCallback.oncomplete (fs.js:169:21) {
  code: 'MODULE_NOT_FOUND'
}
```

Version: 0.8.0
Installation: `yarn global add @metamask/snaps-cli`

**Comment by rekmarks** [comment]
@Vid201 thank you for your bug report. Do you mind sharing:

- Your operating system and version
- Your `yarn` installation method (e.g. homebrew on macOS, a Linux package manager, using `npm install -g` etc.)

Meanwhile, you can create a new repository from the [snaps template repository](https://github.com/MetaMask/template-snap), clone it, and invoke `@metamask/snaps-cli` via `yarn mm-snap <command> [options]`.

**Comment by Vid201** [comment]
Sure.

Operating system: elementary OS 6.1 Jólnir
Installation method: `sudo snap install node --classic` (`yarn` comes with `node`)

Thanks for the link.

**Comment by rekmarks** [comment]
I just realized that this is likely occurring due to `babelify` and its dependencies not being included in the CLI bundle, because `babelify` is never explicitly imported, only referred to using a string literal parameter that Browserify uses to dynamically import it. Fix incoming.

**Comment by rekmarks** [comment]
@Vid201 are there `@babel` and `babelify` directories if you check in `/home/vid/.nvm/versions/node/v12.11.0/lib/node_modules/@metamask/snaps-cli/node_modules/`?

**Comment by Vid201** [comment]
@rekmarks Just checked, they are.

**Comment by rekmarks** [comment]
@Vid201 the fix for this published in `0.9.0`. If you're still having issues, please let us know!

**Comment by Vid201** [comment]
@rekmarks Just checked, it is working now. Tnx!
